select.scss

```diff
@@ -0,0 +1,127 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+
+// Whether to draw the default down-arrow icon.
+$select__draw-caret: true !default;
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/triangle';
+@import 'level/level';
+
+
+// Root
+// ---
+
+.c-select {
+    position: relative; // Allow positioning of select element and caret
+
+    display: block;
+    box-sizing: border-box;
+    max-width: 100%;
+    min-height: $tap-size;
+    border: 1px solid $border-color;
+
+    border-radius: $border-radius;
+
+    font-family: inherit;
+
+    user-select: none;
+
+
+    // Caret
+    // ---
+
+    @if ($select__draw-caret) {
+        &::after {
+            @include triangle(
+                $type: 'triangle',
+                $direction: 'down',
+                $size: 7px,
+                $color: $border-color
+            );
+
+            content: '';
+
+            position: absolute;
```

Comment:
I'm not sure. I've grown into the habit of explaining all positioning, since sometimes they aren't as obvious as they might appear. Maybe I'm going overkill, but better safe than sorry?